docs/admonitions.rst

```diff
@@ -229,10 +231,27 @@ Any of the above builtin admonitions' icons can be changed using the
 .. admonition:: ``seealso`` uses the ``note`` icon
    :class: missing
 
-   The `seealso` admonition (which is specific to Sphinx - not reStructuredText) will use the same
-   icon set for the `note` admonition. If you want to override the icon for the `seealso`
-   admonition, then use the tactic shown in the `Custom admonitions`_ section (with regard to
-   only the icon changes in CSS).
+   The `seealso` admonition (which is specific to Sphinx - not reStructuredText or mkdocs-material
+   theme) will use the same icon set for the `note` admonition. If you want to override only the
+   icon for the `seealso` admonition, then use the tactic shown in the `Custom admonitions`_
+   section (with regard to only the icon changes in CSS). However, instead of copying the SVG data
+   into a CSS variable, one could simply use :themeconf:`icon`\ [:themeconf:`admonition`] field.
+
+   .. code-block:: python
+
+      html_theme_options = {
+          "icon": {
+              "admonition": {
+                  "seealso": "octicons/eye-24", # (1)!
+              },
+          },
+      }
+
+   .. code-annotations::
+      #. Uses the icon :md-icon:`octicons/eye-24`
+
+   Then, use the the generated CSS variable :css:`--md-admonition-icon--seealso` in the custom CSS for
```

Comment:
I'm not sure I understand the intent of this section ---- what is the reason for giving special attention to the `seealso` admonition --- and why is there a mention of CSS variables, etc.?

Reply:
I could make it more generic in saying that

> Some admonitions inherit their styles from the note admonition. This specifically happens for the seealso admonition, which has no specific rules in CSS.

The reason for mentioning the CSS variable is a broad stroke of the implementation in the HTML template. There's no need to copy the SVG data to a custom CSS variable if the desired SVG ships with the theme.

I didn't alter the example CSS about custom admonitions because it may still be useful to demonstrate using SVG data that doesn't ship with the theme.

Reply:
Oh I see --- the issue is that for `seealso` it is not sufficient to just specify an icon under `html_theme_options` because those only work for the admonitions specified in `src/assets/stylesheets/main/extensions/markdown/_admonition.scss`

Ideally we could fix that and add a nice Python interface for defining custom admonitions, but that is independent of this issue.